springwolf-plugins/springwolf-sns-plugin/src/main/java/io/github/stavshamir/springwolf/asyncapi/scanners/channels/operationdata/annotation/SnsAsyncOperationBinding.java

```diff
@@ -20,4 +20,10 @@
 public @interface SnsAsyncOperationBinding {
 
     String type() default "sns";
+
+    String protocol();
+
+    SnsAsyncOperationBindingIdentifier endpoint();
```

Comment:
Both `protocol` and `endpoint` are breaking changes. Documentation needs to be updated.